website_sale_wishlist/models/wishlist_product.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openerp import api, fields, models
+from openerp.http import request, root
+
+
+class WishlistProduct(models.Model):
+    _name = "wishlist.product"
```

Comment:
Possibly `product.wishlist` instead of creating a namespace for wishlists? I can't think of another use for a wishlist outside of products, and this would match pricelist naming